crates/uv-cli/src/lib.rs

```diff
@@ -2037,6 +2037,8 @@ pub enum ToolCommand {
     List(ToolListArgs),
     /// Uninstall a tool.
     Uninstall(ToolUninstallArgs),
+    /// Ensure that the tool executable directory is on `PATH`.
+    Ensurepath,
```

Comment:
I'm not in love with this command name (though I think we should support it via alias regardless). Are there other tools that use this? Any alternatives jump to mind? e.g. `update-shell`?

Reply:
Riffing: `update-path`? `add-path`? `set-path`?

`update-shell` seems ok too. Should it be top-level...?

Reply:
Yeah idk lots of possibilities, also `install-path`. A quick google search made `ensurepath` look pretty `pipx`-specific.

Top-level like `uv update-shell`? 

Perhaps to help inform both: Are we expecting to install other things?

Reply:
pipx uses ensurepath https://fig.io/manual/pipx/ensurepath